backend/pkg/helm/charts.go

```diff
@@ -0,0 +1,74 @@
+package helm
+
+// import (
```

Comment:
There's a TODO here, with code commented out.

Reply:
is this whole file needed? Seems just like a bunch of commented out code...
Where is the TODO?

Reply:
The comment is 'wip'. I don't know why it's here.

Reply:
We were unsure if the charts list endpoint will be required or not but turns out this is required for the release upgrade.